test/unit/helpers/is-interactive-element-test.js

```diff
@@ -41,6 +41,15 @@ describe('isInteractiveElement', function() {
     '<label></label>': '<label>',
     '<div role="button"></div>': 'an element with `role="button"`',
     '<div role="textbox"></div>': 'an element with `role="textbox"`',
+    '<div role="combobox"></div>': 'an element with `role="combobox"`',
+    '<div role="grid"></div>': 'an element with `role="grid"`',
+    '<div role="listbox"></div>': 'an element with `role="listbox"`',
+    '<div role="menu"></div>': 'an element with `role="menu"`',
+    '<div role="menubar"></div>': 'an element with `role="menubar"`',
+    '<div role="radiogroup"></div>': 'an element with `role="radiogroup"`',
+    '<div role="tablist"></div>': 'an element with `role="tablist"`',
+    '<div role="tree"></div>': 'an element with `role="tree"`',
```

Comment:
```suggestion
```
An element with this role can contain interactive elements but is not itself interactive.